.gitignore

```diff
@@ -139,3 +139,5 @@ cython_debug/
 
 # VSCode
 .vscode/
+*.code-workspace
```

Comment:
Q: braucht es diesen change? Willst du neu diese Art files ignorieren?

Reply:
ok, habs wieder mit drin

Reply:
*.code-workspace
 und 
 *.ps1

 Sind immer noch neu im gitignore file. Ist auch nicht falsch das zu ignorieren, ich brauchte das bisher einfach nicht. Ist vermutlich eine Einstellung in deinem VS code?